Wrappers/Python/ccpi/viewer/CILViewer2D.py

```diff
@@ -1555,6 +1553,9 @@ def installPipeline(self):
         self.ren.ResetCamera()
         self.ren.Render()
 
+        self.camera.SetViewUp(0,-1,0)
+        self.camera.Azimuth(180)
```

Comment:
I suppose this should go in `AdjustCamera`?

Reply:
This only happens in `installPipeline`, to make sure the initial camera direction is correct.
Whereas `AdjustCamera` is called in `updatePipeline` as well, where we don't want to move the camera to the xy plane